src/viewer/CornerstoneViewport.js

```diff
@@ -13,6 +13,7 @@ import LoadingIndicator from '../shared/LoadingIndicator.js';
 import Labelling from '../labelling/labelling.js';
 import './CornerstoneViewport.css';
 import guid from './lib/guid.js';
+import _ from 'underscore';
```

Comment:
If you only need isEqual, import that specific thing from lodash: https://www.npmjs.com/package/lodash.isequal